pkgs/development/libraries/mesa/default.nix

```diff
@@ -124,44 +125,43 @@ stdenv.mkDerivation {
     "localstatedir=\${TMPDIR}"
   ];
 
-  # move gallium-related stuff to $drivers, so $out doesn't depend on LLVM;
-  #   also move libOSMesa to $osmesa, as it's relatively big
-  # ToDo: probably not all .la files are completely fixed, but it shouldn't matter
-  postInstall = with stdenv.lib; ''
+  # TODO:
+  #  move gallium-related stuff to $drivers, so $out doesn't depend on LLVM;
+  #  move libOSMesa to $osmesa, as it's relatively big;
+  #  probably not all .la files are completely fixed, but it shouldn't matter;
+  postInstall = ''
```

Comment:
This change obscures the meaning of the comments. The first two lines *are done* by the commands below; only the last one should be TODO.